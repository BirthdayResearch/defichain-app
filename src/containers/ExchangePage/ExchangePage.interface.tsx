export interface ExchangePageProps {}
export interface ExchangePageState {}
