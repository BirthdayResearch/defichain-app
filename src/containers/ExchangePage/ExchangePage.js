import React, { Component } from 'react';
import { Helmet } from "react-helmet";

class ExchangePage extends Component {
  render() {
    return (
      <div className="main-wrapper">
        <Helmet>
          <title>Exchange â€“ DeFi Blockchain Client</title>
        </Helmet>
        <header className="header-bar">
          <h1>Exchange</h1>
        </header>
        <div className="content">
          <section>
            Section
          </section>
        </div>
        <footer className="footer-bar">
          Footer Bar
        </footer>
      </div>
    );
  }
}

export default ExchangePage;