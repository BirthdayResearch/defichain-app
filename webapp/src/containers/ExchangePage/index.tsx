import React from 'react';
import { Helmet } from 'react-helmet';
import { I18n } from 'react-redux-i18n';
import { getPageTitle } from '../../utils/utility';
import Header from '../HeaderComponent';

const ExchangePage = () => {
  return (
    <div className='main-wrapper'>
      <Helmet>
        <title>{getPageTitle(I18n.t('containers.exchangePage.title'))}</title>
      </Helmet>
      <Header>
        <h1>{I18n.t('containers.exchangePage.exchange')}</h1>
      </Header>
      <div className='content'>
        <section>{I18n.t('containers.exchangePage.section')}</section>
      </div>
      <footer className='footer-bar'>
        {I18n.t('containers.exchangePage.footerBar')}
      </footer>
    </div>
  );
};

export default ExchangePage;
