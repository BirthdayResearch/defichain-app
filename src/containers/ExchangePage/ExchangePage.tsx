import React, { Component } from "react";
import { Helmet } from "react-helmet";
import { ExchangePageProps, ExchangePageState } from "./ExchangePage.interface";

class ExchangePage extends Component<ExchangePageProps, ExchangePageState> {
  render() {
    return (
      <div className="main-wrapper">
        <Helmet>
          <title>Exchange â€“ DeFi Blockchain Client</title>
        </Helmet>
        <header className="header-bar">
          <h1>Exchange</h1>
        </header>
        <div className="content">
          <section>Section</section>
        </div>
        <footer className="footer-bar">Footer Bar</footer>
      </div>
    );
  }
}

export default ExchangePage;
