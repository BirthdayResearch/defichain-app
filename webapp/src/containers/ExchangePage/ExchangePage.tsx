import React, { Component } from "react";
import { Helmet } from "react-helmet";
import { ExchangePageProps, ExchangePageState } from "./ExchangePage.interface";
import { I18n } from "react-redux-i18n";

class ExchangePage extends Component<ExchangePageProps, ExchangePageState> {
  render() {
    return (
      <div className="main-wrapper">
        <Helmet>
          <title>{I18n.t("containers.exchangePage.title")}</title>
        </Helmet>
        <header className="header-bar">
          <h1>{I18n.t("containers.exchangePage.exchange")}</h1>
        </header>
        <div className="content">
          <section>{I18n.t("containers.exchangePage.section")}</section>
        </div>
        <footer className="footer-bar">
          {I18n.t("containers.exchangePage.footerBar")}
        </footer>
      </div>
    );
  }
}

export default ExchangePage;
