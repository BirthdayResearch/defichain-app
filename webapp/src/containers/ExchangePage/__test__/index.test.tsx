import React from 'react';
import { shallow } from 'enzyme';
import ExchangePage from '../index';

describe('ExchangePage Page', () => {
  it('should check for snapshot', () => {
    const wrapper = shallow(<ExchangePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
