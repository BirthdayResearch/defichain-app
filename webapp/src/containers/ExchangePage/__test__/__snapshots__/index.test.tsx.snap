// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExchangePage Page should check for snapshot 1`] = `
<div
  className="main-wrapper"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Exchange - DeFi Wallet
    </title>
  </HelmetWrapper>
  <Connect(Header)>
    <h1>
      Exchange
    </h1>
  </Connect(Header)>
  <div
    className="content"
  >
    <section>
      Section
    </section>
  </div>
  <footer
    className="footer-bar"
  >
    Footer bar
  </footer>
</div>
`;
